gitpod-io/gitpod
# [code] Update code to 1.61

## Description
Updates code to 1.61
https://github.com/gitpod-io/openvscode-server/commit/bc3ac84b6b50a15dbcf845bdab46141ffbf8364e

## Related Issue(s)

Fixes #6000

## How to test

1. Switch to latest VS Code in settings.
2. Start a workspace.

Test following:
- terminals are preserved and resized properly between window reloads
- extension host process: check language smartness and debugging
- extension management (installing/uninstalling)
  - install VIM extension to test web extensions
- that user data are sync across workspaces as well as on workspace restart, especially for extensions
  - extensions from .gitpod.yml are not installed as sync
  - extensions installed as sync are actually sync in new workspaces
- settings should not contain any mentions of MS telemetry
- websockets and workers are properly proxied
  - diff editor should be operatable
  - trigger reconnection with window.WebSocket.disconnectWorkspace(), check that old websockets are closed and new opened of the same amount
- workspace specific commands should work, i.e. F1 → type Gitpod prefix
- that a PR view is preloaded on the PR url
- test gp open and preview
- test open in VS Code Desktop, check gp open and preview in task/user terminals

## Release Notes
```release-note
NONE
```
